docs/examples/use_cases/pytorch/single_stage_detector/src/coco_pipeline.py

```diff
@@ -14,77 +14,76 @@
 
 
 import torch
-from nvidia.dali.pipeline import Pipeline
+from nvidia.dali.pipeline import pipeline
```

Comment:
same as in the previous file, do we want to replace our examples to use this?

Reply:
I'm replacing subset of examples